README.md

```diff
@@ -44,12 +44,33 @@ const loadmill = require('loadmill')({token: process.env.LOADMILL_API_TOKEN});
 const result = await loadmill.runTestSuite({id: "test-suite-uuid"});
 ```
 
-You can also extend the suite object:
+You can also extend the suite object with `options` object - containing:
+* additionalDescription - added at the end of the test suite description.
+* labels - will execute only flows attached to these labaels.
+
+Also, you may add a second argument if you wish to override suite parameters
 ```js
 const result = await loadmill.runTestSuite({
     id: "test-suite-uuid",
-    additionalDescription: "description to add", //optional
-    labels: ["label1", "label2"] //optional - run flows that are assigned to specific label/s
+    options: {
+        additionalDescription: "description to add", //optional
+        labels: ["label1", "label2"] //optional - run flows that are assigned to specific label/s
+    },
+    {
+        "parameterKey": "overrided value"
+    }
+}
+```
+
+You can also use one API call to launch all of the team's test suites which have flows marked for execution with the CI toggle:
+```js
+const result = await loadmill.runAllExecutableTestSuites({
+    {
```

Comment:
options object is missing no ?

Reply:
We are passing only the options object. The `.d.ts` is this: 
```runAllExecutableTestSuites(options?: Loadmill.TestSuiteOptions | undefined, params?: Loadmill.Params | undefined, testArgs?: Loadmill.Args | undefined): Promise<Loadmill.TestResult[]>;```